features/liquidity/components/ManagePoolDialog/usePoolDialogController.tsx

```diff
@@ -12,6 +9,7 @@ import {
   UpRightArrow,
   Valid,
 } from 'junoblocks'
+import { PoolEntityTypeWithLiquidity } from 'queries/useQueryPools'
```

Comment:
I'm a little confused what PoolEntityType means? I think we can use a clear name?

Reply:
Open to suggestions but PoolEntity is an entity of a pool and the pool in this case is what we have in the pool list json